src/routes/console/project-[project]/settings/+page.svelte

```diff
@@ -14,11 +14,15 @@
     import { base } from '$app/paths';
     import { page } from '$app/stores';
     import { Submit, trackEvent, trackError } from '$lib/actions/analytics';
+    import EnableAllServices from './enableAllServices.svelte';
+    import DisableAllServices from './disableAllServices.svelte';
 
     let name: string = null;
     let showDelete = false;
     const endpoint = sdk.forConsole.client.config.endpoint;
     const projectId = $page.params.project;
+    let showDisableAll = false;
+    let showEnableAll = false;
```

Comment:
we should probably do reactive statements on those and have state manage them by itself 👍🏻

Reply:
I don't understand what you mean? This is for showing the confirmation dialog?

Reply:
This is how it is implemented for Delete Project using `showDelete` ?